healthtools/scrapers/clinical_officers.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
             "name", "reg_date", "reg_no", "valid_dates",
             "address", "qualifications", "id",
         ]
-
+        self.es_index = ES["index"]
```

Comment:
@DavidLemayian I did this because i had to separate the test index from the dev index 😬